docs/src/configuration/app/multi-app.md

```diff
@@ -179,3 +179,16 @@ There is no requirement that an application be web-accessible.  If it is not spe
 In a multi-app configuration, applications by default cannot access each other.  However, they may declare a `relationships` block entry that references another application rather than a service.  In that case the endpoint is `http`.
 
 However, be aware that circular relationships are not supported.  That is, application A cannot have a relationship to application B if application B also has a relationship to application A.  Such circular relationships are usually a sign that the applications should be coordinating through a shared data store, like a database, [RabbitMQ server](/configuration/services/rabbitmq.md), or similar.
+
+Example:
+
+You have 2 applications, `app1` and `app2`. `app1` wants to connect to `app2` (i.e. `app2` is an API).
+In `app1/.platform.app.yaml` you can make a relationship to `app2` like this:
+
+```
+relationships:
+  app2: "app2:http"
+```
+
+Once committed and rebuilt, you will be able to access `app2` from `app1` via this url `http://app2.internal`. (e.g `curl http://app2.internal`). 
+*Note: the `http`. The relationship is created within the internal network over port 80, not https supported.*
```

Comment:
```suggestion
Once committed and rebuilt, you will be able to access `app2` from `app1` over HTTP using the `host` key of the `app2` relationship in the `PLATFORM_RELATIONSHIPS` variable.

{{< note >}}
Note: the `http`. The relationship is created within the internal network over port 80.  HTTPS connections are not supported.
{{</ note >}}
```

I think that note syntax is right; @chadwcarlson, please check me.

Reply:
Yeah, that's right.